src/js/components/DataTable/doc.js

```diff
@@ -352,6 +352,45 @@ export const themeDoc = {
     type: 'React.Element',
     defaultValue: undefined,
   },
+  'dataTable.pinned.body.background': {
+    description: `The background color for pinned footer cells. Any valid Box 
+    background options apply.`,
+    type: `string | { color: { dark: string, light: string }, dark: bool | 
+    string, image: string, position: string, opacity: string | { dark: string, 
+    light: string }, repeat: no-repeat | repeat, size: cover | contain }`,
```

Comment:
Can we double check the types here? Something looks jumbled up here. Bracket or pipe out of place?

I feel like the type looks something more like this, but could be wrong:

```suggestion
    type: `string | 
               { dark: string, light: string } |
               { 
                   color: { dark: string, light: string }, 
                   dark: bool, 
                   image: string, 
                   position: string, 
                   opacity: bool | string, 
                   repeat: no-repeat | repeat, 
                   size: cover | contain | string
               }`,
```

Reply:
Thank you, good catch I had not included the `{ dark: string, light: string }` option on its own.